src/content/blog/welcome-to-the-new-site.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Welcome to the new site!"
+description: "An introduciton to my new portfolio, and the technology that powers it."
+# watch out date is in standard time shows up wrong fix it
+pubDate: "2024-04-19"
```

Comment:
Dates are coerced into UTC format instead of my timezone, hence why the date is one day ahead.

I can just add the time zone to this and this should work.

Reply:
I should also add a note to do this in my README so I don't forget.